internal/cli/start.go

```diff
@@ -95,6 +96,11 @@ The validate command expects a configuration file as its only argument. Local fi
 
 func Start() error {
 	toolName := "kairos"
+
+	cli.VersionPrinter = func(cCtx *cli.Context) {
```

Comment:
docs: https://cli.urfave.org/v2/examples/version-flag/

Reply:
I would add a specific `version` command so we have the same all over. IIRC kairos-agent+immucore both have a `version` command that prints the version with an extra flag to print the long version:

```
$ go run main.go version        
v0.0.1

# itxaka @ zeus in ~/projects/kairos-agent on git:small_fix_chroot_stages o [12:21:10] 
$ go run main.go version --long
{Version:v0.0.1 GitCommit:none GoVersion:go1.20}
```

And if you use fips it even says it on there:

```
$ GOEXPERIMENT=boringcrypto go run main.go version --long 
{Version:v0.0.1 GitCommit:none GoVersion:go1.20 X:boringcrypto}
```

Reply:
I added `version` subcommand and then realized there was already `--version` and switched to that. I'll switch back to the subcommand to be consistent with the rest of the binaries.

Reply:
done